mmdet/datasets/base_video_dataset.py

```diff
@@ -0,0 +1,299 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+import os.path as osp
+from collections import defaultdict
+from typing import Any, List, Tuple
+
+from mmengine.dataset import BaseDataset
+from mmengine.fileio import FileClient
+from mmengine.logging import print_log
+
+from mmdet.datasets.api_wrappers import COCO
+from mmdet.registry import DATASETS
+
+
+@DATASETS.register_module()
+class BaseVideoDataset(BaseDataset):
```

Comment:
Seems BaseVideoDataset is force set coco style as default. Can we rename it to `CocoVideoDataset`, and add a pure BaseVideoDataset?

Reply:
All the ann files in tracking are coco style after the dataset conversion.